Server/app.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express')
+const app = express();
+const cors = require('cors');
+const path = require('path');
+
+app.use(cors());
+app.use(express.static(path.join(__dirname, 'public')));
+
+// app.get('/', (req, res) => {
+//     res.send('Hello this is Googlarachi!');
+// });
+
+//array of webpages
+const webpages = [
+    "https://www.indomie.co.uk/", "https://www.samyangfoods.com/eng/index.do", "https://www.samyangfoods.com/eng/index.do", "https://www.koreafoods.co.uk/brand/ottogi/",
+     "http://eng.nongshim.com/main/index", "https://nissinnoodles.co.uk/collections/ramen-noodles", "https://www.primataste.com/recipes/laksa-lamian", 
+     "https://www.orientalmart.co.uk/bai-jia-chongqing-burning-dry-noodles-hot-and-sour-flavour", 
+     "https://nongshimusa.com/homev2/bowl-noodle/chapaguri/", "https://www.potnoodle.com/"
+];
```

Comment:
nice deceleration, could use objects if you wanted more dynamic calls from fetch - such as website name, link, and descriptive text on the search page